src/components/AudioDetail.vue

```diff
@@ -349,6 +357,30 @@ export default defineComponent({
     const query = computed(() => audioItem.value?.query);
     const accentPhrases = computed(() => query.value?.accentPhrases);
 
+    const supportedFeatures = computed(
+      () =>
+        store.state.engineManifests[
+          audioItem.value.engineId ?? store.state.engineIds[0]
+        ]?.supportedFeatures || ({} as EngineManifest["supportedFeatures"])
```

Comment:
テキスト欄を追加した後消去するとここでエラーになりました！
消去する直前にaudioItem.valueがundefinedになるっぽいです。。
こうすると動きそう？

```suggestion
        store.state.engineManifests[
          audioItem.value?.engineId ?? store.state.engineIds[0]
        ]?.supportedFeatures || ({} as EngineManifest["supportedFeatures"])
```

Reply:
修正しました。